### ISSUE TITLE
User menu is not accessible by keyboard

### ISSUE THREAD

**Issue by HRemonen** [UniversityOfHelsinkiCS/palaute#1266]
The user menu can be somewhat navigated with keyboard when using a screen reader. However it's still bad. 

2.1.1 (AA) All [functionality](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html#dfn-functionality) of the content is operable through a [keyboard interface](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html#dfn-keyboard-interface) without requiring specific timings for individual keystrokes, except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints. https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

![image](https://github.com/UniversityOfHelsinkiCS/palaute/assets/81435393/f788f1f2-42fc-47bb-a39b-8a817dbf632b)
